src/main/java/com/google/api/codegen/nodejs/NodeJSGapicContext.java

```diff
@@ -379,21 +380,19 @@ private String returnTypeComment(Method method, MethodConfig config) {
    */
   public List<String> methodComments(Interface service, Method msg) {
     MethodConfig config = getApiConfig().getInterfaceConfig(service).getMethodConfig(msg);
-
     // Generate parameter types
     StringBuilder paramTypesBuilder = new StringBuilder();
+    if (config.getRequiredFields().iterator().hasNext()) {
```

Comment:
Hmm, I think this is not right, considering some method whose request does not have any required fields but many optional fields. Possibly `config.getRequiredFields().iterator().hasNext() || config.getOptionalFields().iterator().hasNext())` might be a better condition (a bit verbose though).